src/sentry/lang/native/symbolicator.py

```diff
@@ -254,6 +257,12 @@ def get_sources_for_project(project):
 
     sources = []
 
+    if not getattr(project, "_organization_cache", False):
+        with sentry_sdk.start_span(op="lang.native.symbolicator.organization.get_from_cache"):
+            project._organization_cache = Organization.objects.get_from_cache(
+                id=project.organization_id
+            )
```

Comment:
Should we perhaps do this earlier and right in `symbolicate_event`, just in case something else uses that cache?

Reply:
I am not sure. The idea sounds good but the whole `symbolicate_event` task operates only on `project_id` so we would need to retrieve the project and organisation just for the sake of caching and the retrieval and use will not be together.

imho if something else is using organisation, or any other object that can be cached, should check the cache before use.

Reply:
do we use project.organization in here?

Reply:
We use `project.organization` in at least two places, links are in PR description.